code/obj/item/storage/backpack_belt_etc.dm

```diff
@@ -200,14 +200,6 @@
 		if(!can_use())
 			boutput(user, "<span class='alert'>You need to wear [src] for that.</span>")
 			return
-		if (istype(W, /obj/item/storage/toolbox) || istype(W, /obj/item/storage/box) || istype(W, /obj/item/storage/belt))
```

Comment:
Add `  return ..()` here to call the parent.

Reply:
this deleted bruv

Reply:
?

Reply:
I know it's deleted but without `return ..()` this code won't work so I'm suggesting adding `return ..()` in place of the deleted code.

Reply:
ahh yep!